{"name": "Проверка, что нельзя создать двух одинаковых курьеров.", "status": "passed", "description": "Успешное создание одного курьера, попытка создания второго курьера с теми же учётными данными,проверка получения статуса кода 409 и сообщения об ошибке.", "start": 1734210146365, "stop": 1734210147047, "uuid": "ab0b3275-8431-4c1c-9dba-8eed630d59ad", "historyId": "6832e863dd6d182e4787cc90d059c7da", "testCaseId": "6832e863dd6d182e4787cc90d059c7da", "fullName": "tests.couriers.test_create_courier.TestCreateCourier#test_create_two_couriers", "labels": [{"name": "parentSuite", "value": "tests.couriers"}, {"name": "suite", "value": "test_create_courier"}, {"name": "subSuite", "value": "TestCreateCourier"}, {"name": "host", "value": "SergeiBazhenov"}, {"name": "thread", "value": "12400-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.couriers.test_create_courier"}]}